### PR TITLE
 Add Spectrum Junior Software Backend Developer

### PR DESCRIPTION
**Pull request by piyushkanadje**

### CODE REVIEW

File: README.md
Comment (BrandoZhang) [inline README.md:1]: FYI, this is not Charter Spectrum.

Comment (piyushkanadje) [reply README.md:1]: Which one it is?

Comment (BrandoZhang) [reply README.md:1]: I mean, `https://www.spectruminc.com/careers/` seems not the Spectrum company I am aware of, which is an ISP and owns the domain `www.spectrum.com`. I am afraid of the confusion.

Comment (rajs80266) [reply README.md:1]: @piyushkanadje I feel this is not the job posting of spectrum. They have separate careers page link from the one you mentioned. https://jobs.spectrum.com/
The one you shared https://www.spectruminc.com/careers/detail/?id=798192808143717787 may be some other company.

Comment (piyushkanadje) [reply README.md:1]: Okay I will change the README.

**Review by ReaVNaiL — APPROVED** [review]

**Pull request merged**